packages/proxy-fetcher/src/index.ts

```diff
@@ -82,25 +82,16 @@ export const getFetcher = (
         ...options,
       })
 
-      // Log request failures:
-      //  In debug mode: always
-      //  In non-debug mode: only on final attempt
-      const logFailedRequest = debug ? true : attempt === retryCount + 1
-
       if (!response.ok) {
-        if (logFailedRequest) {
-          console.error(
-            `Failed request (Attempt ${attempt} of ${retryCount + 1}): ${JSON.stringify(
-              {
-                url: response.url,
-                status: response.status,
-                statusText: response.statusText,
-              },
-              null,
-              2
-            )}`
-          )
-        }
+        const logOrError = attempt <= retryCount ? log : error
+        logOrError(
+          `Failed request (Attempt ${attempt} of ${retryCount + 1}): %o`,
+          {
+            url: response.url,
+            status: response.status,
+            statusText: response.statusText,
+          }
+        )
```

Comment:
Looks like this now:
![image](https://github.com/user-attachments/assets/0cea987d-a78f-4ce0-85c5-d7e911af2676)

If you don't want to see all the logs for all the failed attempts, you can add `-proxy-fetcher:log` to the `DEBUG` env var.

It's kind of a bummer that I get a `+0ms` on that last error instead of `+16s`, but 🤷 .